### PR TITLE
Admin can publish results from dashboard

### PR DESCRIPTION
**Pull request by holgertidemand**
#### PT Story: https://www.pivotaltracker.com/story/show/156430926
Changes proposed in this pull request:

*Adds publish result button to admin dashboard

#### What I have learned working on this feature: 
* Working with proper class methods
* Better understanding of monkey patching even tho we didn't use it in the end
#### Screenshots:
screen shot 2018-04-10 at 13 02 39

### CODE REVIEW

File: features/admin_can_publish_result.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Feature: Admin can publish result
+  As an admin
+  In order to inform my end users the result of the competition
+  I would like to be able to share the results.
+
+
```
Comment (davidboll) [inline features/admin_can_publish_result.feature]: Remove empty line

**Review by davidboll — CHANGES_REQUESTED** [review]

File: app/controllers/athletes_controller.rb [hunk 2]
```diff
@@ -18,11 +18,21 @@ def create
   end
```
Comment (davidboll) [inline app/controllers/athletes_controller.rb]: Remove line

File: app/controllers/athletes_controller.rb [hunk 3]
```diff
@@ -18,11 +18,21 @@ def create
...
+  def publish
+    if Result.publish_results
+      flash[:success] = 'Result successfully published'
+      redirect_to root_path
+    else
+      flash[:error] = 'You have no results to publish'
+      redirect_to root_path
+    end
+  end
   private
```
Comment (davidboll) [inline app/controllers/athletes_controller.rb]: Add line

**Review by davidboll — CHANGES_REQUESTED** [review]

**Review by davidboll — APPROVED** [review]

**Review by larathordar — APPROVED** [review]
Looks good !

**Pull request merged**